test/conformance/tests/test_policy_configuration.go

```diff
@@ -0,0 +1,790 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	sriovv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/clean"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/cluster"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/discovery"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/namespaces"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/network"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/pod"
+)
+
+var _ = Describe("[sriov] operator", Ordered, func() {
+	var sriovInfos *cluster.EnabledNodes
+
+	BeforeAll(func() {
+		isSingleNode, err := cluster.IsSingleNode(clients)
```

Comment:
this is the same as in the other file can we just have an internal function that will call BeforeAll?

Reply:
I refactored to a common `BeforeSuite` section. It's the best we can do IMO because these BeforeAll sections modify shared variables. WDYT?